src/about.md

```diff
@@ -6,19 +6,18 @@ eleventyNavigation:
   key: About
   order: 3
 ---
-## About PolyMC
+# About PolyMC
 
-PolyMC is a custom launcher for Minecraft forked from MultiMC that allows to manage more instances at once.
-While this fork was made for packaging/redistribution concerns, now this fork has a lot of more features developed by the community for the community.
-In fact, unlike other launchers, PolyMC has pure freedom about redistribution: it has reproducible builds, it allows everyone to package it, and we don't care if you fork (if you respect General Public License v3).
+PolyMC is a custom launcher for Minecraft that was forked from MultiMC. It allows the creation, and management of multiple installations of the game, or "instances" rather, each having their own separate mods, reasource packs, settings, and more. While originally being forked over packaging/redistribution concerns, PolyMC brings new and exciting features developed by the community for the community. Furthermore, PolyMC is still just as true to its roots as ever in providing a project that vows to never bring hostility to those looking to fork, redistribute, or repackage, so long as the terms of our license are respected. 
 
-It includes many features, such has:
-- Managing multiple isolated instances of Minecraft,
-- Simple access management about logs/mods and a simple way to install mods from curseforge and modrinth
-- Kill Minecraft easily if it crashes / freezes.
-- Unlike other launchers, PolyMC uses Qt, a simple and lightweight toolkit that doesn't use much system resources
-- Easy installation of mod loaders, such as  Forge, Fabric, LiteLoader and Quilt (WIP).
-- Manage your worlds and resource/texture packs with just some clicks 
-- Install modpacks from many platforms such as Curseforge, Technic, FTB, Modrinth (WIP) and more
-- Install individual mods to match your loader and version from Curseforge and Modrinth.
-- And many, many other features
+## I'm in! Now tell me more about these features?
+
+- Managing multiple, isolated instances of Minecraft.
+- Easy installation of mod loaders, such as  Forge, Fabric, LiteLoader and *Quilt (WIP)*.
```

Comment:
Not so WIP at this point, it's going to be in 1.2.0 at this point

Reply:
(this isn't a blocker but just to say)

Reply:
I'd say it might be worth leaving it for now until quilt is actually implemented in a release build, or we could do something like this? 
```suggestion
- Easy installation of mod loaders, such as  Forge, Fabric, LiteLoader and *Quilt (as of PolyMC 1.2.0)*.
```